sootup.callgraph/src/test/java/sootup/callgraph/CallGraphDifferenceTest.java

```diff
@@ -0,0 +1,70 @@
+package sootup.callgraph;
+
+import java.io.File;
+import java.util.Collections;
+import java.util.List;
+import org.apache.commons.lang3.tuple.Pair;
+import org.junit.jupiter.api.Test;
+import sootup.core.inputlocation.AnalysisInputLocation;
+import sootup.core.signatures.MethodSignature;
+import sootup.java.bytecode.inputlocation.JavaClassPathAnalysisInputLocation;
+import sootup.java.core.types.JavaClassType;
+import sootup.java.core.views.JavaView;
+
+public class CallGraphDifferenceTest {
+
+  @Test
+  public void testCGDiff() {
+    String classPath =
+        System.getProperty("user.dir")
+            + File.separator
+            + "target"
+            + File.separator
+            + "test-classes"
+            + File.separator
+            + "callgraph"
+            + File.separator
+            + "CallGraphDifference"
+            + File.separator
+            + "binary";
```

Comment:
please add .class file to shared test resources - this is heavily relying on the maven /target/ structure and leads to problems if people execute it via gradle-maven integration

Reply:
okay, will make these changes and also update the branch. Thanks ;)